frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -127,6 +128,8 @@ export const defaultBounds = new LatLngBounds(
   [48.78370426, -139.35937554],
 );
 
+var counter = 1000000;
```

Comment:
This is a primitive way to ensure all properties from geocoder receive a unique ID, which is required for some of the existing logic to filter duplicates.